.packit.yaml

```diff
@@ -2,7 +2,7 @@
 # https://packit.dev/docs/configuration/
 actions:
     changelog-entry:
-        - "New upstream release"
+        - bash -c 'echo "- New upstream release"'
```

Comment:
Im not entirely sure, since it's hard to really test it. Looking at their own examples there is a mix of using bash -c 

https://packit.dev/docs/configuration/examples

Reply:
They use `bach -c` when they need `|` or other shell features as far as I can see. This is fine as is.

Reply:
https://packit.dev/docs/configuration/actions (for reference)